tests/marqo_test.py

```diff
@@ -21,6 +21,17 @@
 from marqo.tensor_search.telemetry import RequestMetricsStore
 from marqo.vespa.vespa_client import VespaClient
 
+TEST_IMAGE_URLS = {
```

Comment:
I think an enum for these keys would be great, or a class and one method to get each URL (or a metaclass that generates a class from the list? Probably overkill)

Reply:
Is it redundant to have both an enum for image keys and an object for the URLs? In the tests, we'd just want to fetch the actual URL for downloading, so why don't we just have 1 enum with the URLs? Something like:
```
class TestImageURLs(Enum):
    IMAGE0 = 'https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image0.jpg'
    IMAGE1 = 'https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg'
```

Reply:
In the tests, we could then use `TestImageURLs.IMAGE0` instead of `TEST_IMAGE_URLS[ImageKey.IMAGE0]`

Reply:
Good point. @vicilliar . I'll change that. I'll probably need to use `TestImageURLs.IMAGE0.value` in the test